pycqed/analysis/analysis_toolbox.py

```diff
@@ -873,6 +873,171 @@ def smooth(x, window_len=11, window='hanning'):
     return y[edge:-edge]
 
 
+def find_second_peak(sweep_pts=None, data_dist_smooth=None,
+                     key=None, peaks=None, percentile=20, optimize=False,
+                     verbose=False):
+
+    """
+    Used for qubit spectroscopy analysis. Find the second gf/2 peak/dip based on
+    the location of the tallest peak found, given in peaks, which is the result
+    of peak_finder. The algorithm takes the index of this tallest peak and looks
+    to the right and to the left of it for the tallest peaks in these new
+    ranges. The resulting two new peaks are compared and the tallest/deepest
+    one is chosen.
```

Comment:
Could you add some info on the args and return value to the docstring? 
Otherwise nice function 👍